libshaderc/src/shaderc_test.cc

```diff
@@ -222,8 +222,8 @@ TEST(CompileStringWithOptions, DisassemblyOption) {
   shaderc_compile_options_set_disassembly_mode(options.get());
   ASSERT_NE(nullptr, compiler.get_compiler_handle());
   const std::string kMinimalShader = "void main(){}\n";
-  Compilation comp(compiler.get_compiler_handle(), kMinimalShader,
-                   shaderc_glsl_vertex_shader, options.get());
+  const Compilation comp(compiler.get_compiler_handle(), kMinimalShader,
```

Comment:
This isn't mentioned by the commit comment.
But it's *enabled* by the functional change described in the commit comment (making the result() method const).

I'm ok with this in this CL since it's in test code only.  (No public semantic impact.)  Otherwise I'd prefer a slightly expanded commit comment.